lib/routes/twitter/likes.ts

```diff
@@ -1,15 +1,27 @@
 import { Route } from '@/types';
 import utils from './utils';
-import { config } from '@/config';
-import ConfigNotFoundError from '@/errors/types/config-not-found';
+import api from './api';
 
 export const route: Route = {
     path: '/likes/:id/:routeParams?',
     categories: ['social-media'],
     example: '/twitter/likes/DIYgod',
     parameters: { id: 'username', routeParams: 'extra parameters, see the table above' },
     features: {
-        requireConfig: false,
+        requireConfig: [
+            {
+                name: 'TWITTER_USERNAME',
+                description: 'Please see above for details.',
+            },
+            {
+                name: 'TWITTER_PASSWORD',
+                description: 'Please see above for details.',
+            },
```

Comment:
```suggestion
```
Explained in https://github.com/DIYgod/RSSHub/pull/15384#issuecomment-2093619901